### PR TITLE
Refine the community membership

### PR DESCRIPTION
**Pull request by xiaofan-luan**
Refine the community membership document
Signed-off-by: xiaofan-luan <xiaofan.luan@zilliz.com>

### CODE REVIEW

File: community-membership.md [hunk 1]
```diff
@@ -1,63 +1,99 @@
...
+- They can be assigned to issues and PRs, and people can ask contributors for reviews with a `/assign @username`.
+- Tests can be run against their PRs automatically. 
+- Ask for mentorship from Committers/Maintainers
+- Join the community meetings and give their advice
+
+### Committer
+Commiters are able to review code for quality and correctness on some part of the project. They are knowledgeable about both the codebase and software
+engineering principles and they should pratice the community core values and guidelines as an example.
+
+**Defined by:** [COMITTER](https://github.com/milvus-io/milvus/blob/master/OWNERS) ?具体commiter列表需要重新讨论
```
Comment (yhmo) [inline community-membership.md:46]: not pure english

File: community-membership.md [hunk 2]
```diff
@@ -1,63 +1,99 @@
...
+- Have read the [contributor guide](https://github.com/milvus-io/community/blob/master/CONTRIBUTING.md)
+- Actively contributing to 1 or more milvus related repos.
+
+#### Responsibilities and privileges
+
+- Expected to be responsive to issues and PRs assigned to them
+- Expected to follow community rules and values
+- They can be assigned to issues and PRs, and people can ask contributors for reviews with a `/assign @username`.
+- Tests can be run against their PRs automatically. 
+- Ask for mentorship from Committers/Maintainers
```
Comment (jennyli-z) [inline community-membership.md:39]: how to ask for metorship?

File: community-membership.md [hunk 3]
```diff
@@ -1,63 +1,99 @@
...
+- Nominated by at least 1 Maintainer, seconded by at least 3 other committers/maintainers.
+
+#### Responsibilities and privileges
+- Expected to be responsive to review pull requests
+- Expected to assign issues and prs to related expertise
+- Expected to be responsive to mentions of others on github and slack
+- Expected to join the community meetings and activities in time
+- Committer status may be a precondition to accepting large code contributions
+- Mentor and guide new contributors
+- Committers can do `/lgtm` on open PRs.
```
Comment (jennyli-z) [inline community-membership.md:64]: For now, `/lgtm` is controlled by prow- https://prow.zilliz.cc/command-help#lgtm , all Collaborators on the repository can do.

`/approve` -https://prow.zilliz.cc/command-help#approve-Users listed as 'approvers' in appropriate OWNERS files
Does this mean we need do something to change current situation? @LoveEachDay  please help review if I have any misunderstanding.

File: community-membership.md [hunk 4]
```diff
@@ -1,63 +1,99 @@
...
+- Expected to be responsive to issues and PRs assigned to them
+- Expected to follow community rules and values
+- They can be assigned to issues and PRs, and people can ask contributors for reviews with a `/assign @username`.
+- Tests can be run against their PRs automatically. 
+- Ask for mentorship from Committers/Maintainers
+- Join the community meetings and give their advice
+
+### Committer
+Committers are able to review code for quality and correctness on some part of the project. They are knowledgeable about both the codebase and software
+engineering principles and they should pratice the community core values and guidelines as an example.
```
Comment (LoveEachDay) [inline community-membership.md:44]: typo: practice

File: community-membership.md (hunk 3 above)
Comment (LoveEachDay) [reply community-membership.md:64]: committers can only do `/lgtm`, mainteners can do `/approve`. 
for now, we don't need any changes.

**Comment by SuperSupeng** [comment]
#110

File: community-membership.md [hunk 5]
```diff
@@ -1,63 +1,99 @@
...
-towards the health of that subproject.  Subproject Owners *MUST* set technical
-direction and make or approve design decisions for their subproject - either
-directly or through delegation of these responsibilities.
-
-**Defined by:** *owners* entry in subproject [OWNERS] files as defined by [sigs.yaml]  *subproject.owners*
-
-
+| Contributor | Active contributor in the community | Finish at least one contribution to the project | Github all contributor list|
+| Committer | Review contributions from other members | Finish at least one major contributions, History of reviews| [COMITTER](https://github.com/milvus-io/milvus/blob/master/OWNERS) |
+| Maintainer | Contributions acceptance approval| Highly experienced active reviewer and contributor| [MAINTAINERS](https://github.com/milvus-io/milvus/blob/master/OWNERS_ALIASES)|
```
Comment (SuperSupeng) [inline community-membership.md:12]: I'm a little confused. How do you define Highly experienced active reviewer and contributor？ And what is the "reviewer" role?

Comment (jennyli-z) [reply community-membership.md:12]: I think "committer" is the role you mentioned as "reviewer" role. 
I read the essay https://opensource.guide/leadership-and-governance/ 
A “contributor” could be anyone who comments on an issue or pull request, people who add value to the project (whether it’s triaging issues, writing code, or organizing events), or anybody with a merged pull request (perhaps the narrowest definition of a contributor).
The term “committer” might be used to distinguish commit access, which is a specific type of responsibility, from other forms of contribution.
It seems that we follow the similar definition, right? @xiaofan-luan

**Pull request closed without merging**